Bottom: 15d3bf8cb30548ec2d4a209c49d8311ef518f6dc
Top:    3085b851b213d025aba6499e79497d7f424f05a4
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-28 03:17:18 +0100

hid/gtk: Remove unused extern HID ghid_hid from gui-top-window.c


---

diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index 3d3896f..9fd146e 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -218,8 +218,6 @@ static ToggleFlagType *tflags = 0;
 static int n_tflags = 0;
 static int max_tflags = 0;
 
-extern HID ghid_hid;
-
 GhidGui _ghidgui, *ghidgui = NULL;
 
 GHidPort ghid_port, *gport;
